std/datetime/stopwatch.d

```diff
@@ -349,6 +349,13 @@ private:
     long _ticksElapsed;    // Total time that the StopWatch ran before it was stopped last.
 }
 
+private void doNotOptimizeAway(T)(auto ref T t) @trusted @nogc nothrow
```

Comment:
I think we need to inline this in `benchmark` because otherwise we'll include `getpid` in the measurements.